src/main/java/sirius/biz/tenants/TenantController.java

```diff
@@ -420,7 +420,7 @@ public void selectTenant(final WebContext webContext, String tenantId) {
                                            + TenantUserManager.TENANT_SPY_ID_SUFFIX, null);
             }
 
-            webContext.respondWith().redirectTemporarily("/tenants/select");
```

Comment:
I'd keep the original url if "main" is present, as this is commonly used to "go back to the main tenant and select a new one"